packages/paste-website/src/pages/components/skeleton-loader/index.mdx

```diff
@@ -0,0 +1,233 @@
+import {Anchor} from '@twilio-paste/anchor';
+import {Avatar} from '@twilio-paste/avatar';
+import {Box} from '@twilio-paste/box';
+import {Button} from '@twilio-paste/button';
+import {Card} from '@twilio-paste/card';
+import {Heading} from '@twilio-paste/heading';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {SkeletonLoader} from '@twilio-paste/skeleton-loader';
+import {Stack} from '@twilio-paste/stack';
+import {Table, THead, TBody, Tr, Td, Th} from '@twilio-paste/table';
+import {Text} from '@twilio-paste/text';
+import {CalendarIcon} from '@twilio-paste/icons/esm/CalendarIcon';
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  CompositionExample,
+  TextExample,
+  ParagraphExample,
+  HeadingExample,
+  ButtonsExample,
+  AvatarExample,
+  IconExample,
+  CardExample,
+  TableExample,
+} from '../../../component-examples/SkeletonLoaderExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/skeleton-loader"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/components/skeleton-loader/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Skeleton Loader"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Skeleton Loader"
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/skeleton-loader"
+  storybookUrl="/?path=/story/components-skeleton-loader--default"
+  data={props.data.allPasteComponent.edges}
+  packageStatus={props.data.allAirtable.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Skeleton Loader
+
+Skeleton Loader displays a placeholder of a page's content before the data gets loaded. This helps reduce load-time frustration by making it feel like loading is happening immediately.
+
+It's an alternative to the traditionaly spinner loading methods. Rather than showing an abstract spinner, it creates anticipation of whats to come and reduces congitive load.
+
+The component is designed to be used in place of the content being loaded. For example:
+
+<LivePreview scope={{SkeletonLoader, Text}} noInline>
+  {CompositionExample}
+</LivePreview>
+
+### Accessibility
+
+- The animation can be disabled using the `disabled` prop, or by enabled `prefers-reduced-motion` at the OS level.
+- The animation should be disabled if content takes longer than 5 seconds to load.
```

Comment:
I think we should bake this into the animation loop. People won't do it, and potentially won't know.

We can solve it by just making sure the animation loop is 5 seconds on, and 5 seconds off.

Reply:
Fought with this a bit today, and then realized that the animation was taking almost 5 seconds to fully finish, so I added a small delay before the loop starts.

Reply:
Looks pretty neat actually, I dig it